src/lib/config.js

```diff
@@ -1,20 +1,6 @@
 'use strict'
 
 const Config = require('five-bells-shared').Config
```

Comment:
Also, when calling `Config.loadConfig()` it's natural to call the returned object `config`